spark/spark-spa/src/main/java/com/thoughtworks/go/spark/spa/NewDashboardDelegate.java

```diff
@@ -57,6 +60,7 @@ public ModelAndView index(Request request, Response response) {
             put("viewTitle", "Dashboard");
             put("isQuickEditPageEnabled", Toggles.isToggleOn(Toggles.PIPELINE_CONFIG_SINGLE_PAGE_APP) && Toggles.isToggleOn(Toggles.QUICK_EDIT_PAGE_DEFAULT));
             put("isNewDashboardPageEnabled", Toggles.isToggleOn(Toggles.NEW_DASHBOARD_PAGE_DEFAULT));
+            put("isUserAnAdmin", securityService.isUserAdmin(currentUsername()));
```

Comment:
can we show pipeline level analytics for pipeline group admins as well?

Reply:
The check whether user is admin (for pipeline analytics in GoCD's dashboard) is just a decision made for the purposes of not showing our commercial plugin till it is released. We don't need to call it out as such. We can name it: "shouldShow...".